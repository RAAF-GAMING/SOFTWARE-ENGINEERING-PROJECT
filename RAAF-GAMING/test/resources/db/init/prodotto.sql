CREATE TABLE IF NOT EXISTS prodotto(
  `codice_prodotto` int  NOT NULL AUTO_INCREMENT,
  `prezzo` double NOT NULL,
  `copertina` longblob,
  `sconto` decimal(2,0) DEFAULT '0',
  `data_uscita` date NOT NULL,
  `nome` varchar(50) NOT NULL,
  `quantita_fornitura` int NOT NULL,
  `data_fornitura` date NOT NULL,
  `fornitore` varchar(20) DEFAULT NULL,
  `gestore` varchar(45) DEFAULT NULL,
  PRIMARY KEY (`codice_prodotto`)
  );