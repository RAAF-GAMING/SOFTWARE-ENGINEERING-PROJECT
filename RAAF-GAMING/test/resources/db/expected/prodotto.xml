<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <prodotto codice_prodotto='1' prezzo="10.5" sconto="0" data_uscita="2021-12-25" nome="FIFA" quantita_fornitura="12"
    data_fornitura="2020-12-20" fornitore="Sony" gestore="prodotto@admin.com"/>
    <prodotto codice_prodotto='2' prezzo="15.5" sconto="0" data_uscita="2020-12-22" nome="PES" quantita_fornitura="12"
    data_fornitura="2019-12-20" fornitore="Activision" gestore="prodotto@admin.com"/>
     <prodotto codice_prodotto='3' prezzo="11.0" sconto="0" data_uscita="2021-12-22" nome="WWE2K22" quantita_fornitura="100"
    data_fornitura="2021-02-20" fornitore="2K" gestore="prodotto@admin.com"/>
</dataset>